driver/evrmemmap.h

```diff
@@ -97,12 +97,19 @@ struct DBufInfo {
 /* These must be powers of two!!! */
 #define MAX_EVR_EVTQ 1024
 #define MAX_EVR_DBQ     4
+#define MAX_EVR_DBQ2   16
 struct EvrQueues {
   struct FIFOEvent evtq[MAX_EVR_EVTQ];     /* 12K */
   struct DBufInfo  dbq[MAX_EVR_DBQ];       /*  8K + 16 */
```

Comment:
Is there any way we can make it clear that new applications should be using dbq2 instead of dbq? It's clear to you and to anyone who has seen this PR but not necessarily obvious in general.

Reply:
dbq_new?  I'm open to new names.  Can't change dbq of course... but dbq2 can be whatever we want it to be.

Reply:
Maybe just an "obsolete, use dbq2" comment or some equivalent documentation entry somewhere?